### PR TITLE
IDVA6-1926 Web Security Node Upgrade

### PR DESCRIPTION
**Pull request by kwamenyantakyi-ch**
**JIRA link**
https://companieshouse.atlassian.net/browse/IDVA6-1926

**Description**
Updating **web security node** to 4.4.3 and **node session handler** to 5.2.0

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: .env.example [hunk 1]
```diff
@@ -28,4 +28,4 @@ NODE_HOSTNAME=localhost
 NODE_PORT=3001
 MATOMO_ADD_USER_GOAL_ID=3
 MATOMO_REMOVE_USER_GOAL_ID=4
-COOKIE_SECURE_ONLY=false
\ No newline at end of file
+ENV_NAME=local
```
Comment (mmodzelewski-ch) [inline .env.example:31]: @kwamenyantakyi-ch remember to add this variable to `services/modules/acsp-manage-users/acsp-manage-users-web.docker-compose.yaml` in docker-chs-development repository

**Review by mmodzelewski-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**